src/main/java/org/sagebionetworks/web/client/view/GridPageViewImpl.java

```diff
@@ -0,0 +1,38 @@
+package org.sagebionetworks.web.client.view;
+
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.SimplePanel;
+import com.google.gwt.user.client.ui.Widget;
+import com.google.inject.Inject;
+import org.gwtbootstrap3.client.ui.html.Span;
+import org.sagebionetworks.web.client.widget.entity.SynapseGridImpl;
+
+public class GridPageViewImpl extends Composite implements GridPageView {
+
+  SynapseGridImpl synapseGrid;
+
+  public interface Binder extends UiBinder<Widget, GridPageViewImpl> {}
+
+  @UiField
+  Span gridSessionNameContainer;
+
+  @UiField
+  SimplePanel componentContainer;
+
+  @Inject
+  public GridPageViewImpl(final Binder uiBinder, SynapseGridImpl synapseGrid) {
+    initWidget(uiBinder.createAndBindUi(this));
+
+    this.synapseGrid = synapseGrid;
+
+    componentContainer.add(synapseGrid.asWidget());
+  }
+
+  @Override
+  public void render(String sessionId) {
+    this.gridSessionNameContainer.setText(sessionId);
+    this.synapseGrid.configure(sessionId, false);
+  }
```

Comment:
Super simple-the `sessionId` is passed by the presenter, and we pass it to the React component widget to load the grid.

Reply:
No, it's an opaque string that is not particularly informative for the user. We'll definitely do more work later to display more useful information. This part of the design is totally in flux--consider this a placeholder.